dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -200,12 +200,6 @@ public void testDashboard_statisticBox() {
 
   @Test
   public void testLinkageReports() {
-    Nodes reports = details.query("//p[@class='jar-linkage-report']");
```

Comment:
Why were these assertions deleted? This refactoring shouldn't materially change the output in the dashboard, right?

Reply:
Reverting that part.